resource/translations/ao_es.ts

```diff
@@ -865,17 +866,18 @@ Did you download all resources correctly from tiny.cc/getao, including the large
     <message>
         <location filename="../../src/debug_functions.cpp" line="10"/>
         <source>Error: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Error: %1</translation>
     </message>
     <message>
         <location filename="../../src/debug_functions.cpp" line="11"/>
         <source>Error</source>
-        <translation type="unfinished">Error</translation>
+        <translation>Error</translation>
     </message>
     <message>
         <location filename="../../src/debug_functions.cpp" line="23"/>
         <source>Notice</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>In spanish it would be &quot;Aviso&quot;, but i believe it&apos;s going to be more useful for bug reports to not translate any debug string.</translatorcomment>
```

Comment:
Localized error strings are a common reason to introduce error codes.

Reply:
eww, that's so windows style tho